src/compiler/crystal/syntax/parser.cr

```diff
@@ -3648,6 +3659,24 @@ module Crystal
       {arg_name, external_name, found_space, uses_arg}
     end
 
+    def invalid_internal_name?(keyword)
+      case keyword
+      # These names are handled as keyword by `parser_atomic_without_location`.
```

Comment:
The reference should be `parse_atomic_without_location` (without the `r`). And I would add a note in `Parser#parse_atomic_without_location` as well to keep both list in sync.

Reply:
Thank you. I'll fix it.